src/main/java/com/julienviet/pgclient/impl/PgPoolImpl.java

```diff
@@ -92,6 +92,11 @@ protected void onFailure(Throwable cause) {
 
     protected abstract void onFailure(Throwable cause);
 
+    @Override
+    public void handleNotification(int processId, String channel, String payload) {
+      // What should we do ?
```

Comment:
Maybe route to active connections if they have a notification handler? This also means that returning to the pool it should clear the handlers

Reply:
I think that in a pool, notification should not be used because it is asynchronous